pages/common/jq.md

```diff
@@ -23,7 +23,7 @@
 
 `{{cat path/to/file.json}} | jq '{{.[index1], .[index2], ...}}'`
 
-- Print all array items/object keys:
+- Print all array/object values:
```

Comment:
It should say `object values` rather than `object keys`.

https://jqlang.github.io/jq/manual/#array-object-value-iterator

```
Array/Object Value Iterator: .[] 
If you use the .[index] syntax, but omit the index entirely, it will return all of the elements of an array. Running .[] with the input [1,2,3] will produce the numbers as three separate results, rather than as a single array. A filter of the form .foo[] is equivalent to .foo | .[].

You can also use this on an object, and it will return all the values of the object.
```